.github/workflows/linux_riscv_xuantie.yml

```diff
@@ -0,0 +1,256 @@
+name: Linux RISC-V with Xuantie (Ubuntu 22.04, Python 3.10)
+on:
+  workflow_dispatch:
+    inputs:
+      testFilter:
+        description: 'Filter for google tests'
+        required: true
+        default: '*smoke_AdaPoolAvg4DLayoutTest*'
+  pull_request:
+    paths:
+      - '.github/workflows/dev_cpu_linux_riscv.yml'
```

Comment:
Seems like this is outdated path and there should be `'.github/workflows/linux_riscv_xuantie.yml'`. Could you please take a look?